### PR TITLE
Use acquisition_price in the unconfirmed max price PDF

### PR DESCRIPTION
**Pull request by matti-lamppu**
# Hitas Pull Request

# Description

In unconfirmed max price pdf, _**Alkuperäinen velaton hankintahinta, euroa**_ is now the `acquisition_price` (=`debt_free_purchase_price` + `primary_loan_amount`).

## Pull request checklist

Check the boxes for each DoD items that has been completed:

- **Frontend**
    - [ ] Changes have been tested
- **Backend**
    - [x] Changes have been tested
    - [ ] Automatic tests has been added
    - [ ] OpenAPI definitions have been updated
    - [ ] Database migrations will work in test environment
    - [ ] Oracle migration has been updated
    - [ ] initial.json has been updated to work with migrations

## Test plan

See description.

## Tickets

This pull request resolves all or part of the following ticket(s): HT-360

### CODE REVIEW

File: backend/hitas/views/apartment.py [hunk 1]
```diff
@@ -813,6 +813,12 @@ def download_latest_unconfirmed_prices(self, request, **kwargs) -> Union[HttpRes
             )
 
         sapc = SurfaceAreaPriceCeiling.objects.only("value").get(month=this_month())
+        apartment_data["prices"]["acquisition_price"] = (
+            apartment_data["prices"]["debt_free_purchase_price"] + apartment_data["prices"]["primary_loan_amount"]
+            if apartment_data["prices"]["debt_free_purchase_price"] is not None
+            and apartment_data["prices"]["primary_loan_amount"] is not None
+            else None
+        )
```
Comment (ranta) [inline backend/hitas/views/apartment.py:821]: This same thing is already done in the `PricesSerializer`, so why is this here? 
The value of `acquisition_price` will never be changed by this

**Review by ranta — CHANGES_REQUESTED** [review]

File: backend/hitas/views/apartment.py (hunk 1 above)
Comment (matti-lamppu) [reply backend/hitas/views/apartment.py]: Somehow didn't see this in the serializer, must have been sleeping on the job. Anyway, removed this as it is unnecessary.

**Review by ranta — APPROVED** [review]
🤴
